.gitignore

```diff
@@ -1 +1,11 @@
 node_modules
```

Comment:
Tip for you! 
https://www.gitignore.io/
helps you build git ignores for your ide and for other things based on your language and what not. You would've still needed to append client to all the node specific ones though 🤔